src/components/BarList/BarLabel.tsx

```diff
@@ -0,0 +1,44 @@
+import { FC, ReactNode } from 'react'
+import { composeClasses } from 'lib/classes'
+import { Size } from '../../interfaces/types'
+import Text from '../Typography'
+import Anchor from '../Anchor/Anchor'
+
+export interface BarLabelProps {
+  children: ReactNode
+  fontSizeBar?: Size
+  href?: string
+  textBarColor?: string
+}
+
+export const BarLabel: FC<BarLabelProps> = ({
```

Comment:
I recommend export by default here and in the index.ts named export, like:

BarLabel.ts
```js
export default BarLabel
```

index.ts
```js
export { default as BarLabel } from './BarLabel'